app/views/helpers/lists.php

```diff
@@ -470,7 +470,7 @@ public function displayAddSentenceForm($listId)
                 'class' => 'submit'
             )
         );
-        echo "<div class='loader-container'></div>";
+        echo "<md-progress-circular id='list_add_loader' md-diameter='11px' style='display: none;'></md-progress-circular>";
```

Comment:
All instances. Unless it breaks the UI, in which case, just make it as close as possible to 16px.